Sources/SBTUITestTunnelClient/include/SBTUITestTunnelClientProtocol.h

```diff
@@ -29,6 +29,13 @@
  */
 - (void)launchTunnel;
 
+/**
+ *  Launch application synchronously waiting for the tunnel server connection to be established.
+ *
+ *  @param url URL to launch the app with.
+ */
+- (void)launchTunnelWithURL:(nullable NSURL *)url;
```

Comment:
I would consider renaming the method making it more similar to the XCUIApplication one. What about `openTunnelWithURL`?  The same applies to other methods.

I would also add the availability to all methods specifying the url  by adding the`NS_AVAILABLE_IOS(16.4);` suffix

Reply:
updated! to silence warnings, I had to include `API_AVAILABLE` in .m files and suppressed the warning internally wherever we’re calling a method with a nil URL (let me know if you have a better idea here)

Reply:
Could we also add ` NS_SWIFT_NAME(openTunnel(url:));` to make the API more swifty (and ` NS_SWIFT_NAME(openTunnel(url:startupBlock:));` for the other method)?